recipes/python/python-2.7.11/host-tools.patch

```diff
@@ -0,0 +1,99 @@
+The python build system compiles its own binaries for later
+use during the build. This obviously fails during a cross build,
+so patch the configure script to support defining the pgen tool
+in the environment, and patch the makefile to make use of that
+definition.
+
+Upstream-status: pending
```

Comment:
Homebrewed. Added a line about it being inspired from here:
https://bugs.python.org/file31991/Python-2.7.5-xcompile.patch